cert-manager/cert-manager-kube-system-resources/base/params.env

```diff
@@ -0,0 +1 @@
+certManagerNamespace=cert-manager
```

Comment:
can you use 
namespace=cert-manager 
instead of 
certManagerNamespace=cert-manager
?

Other manifest targets also use namespace. 
In the config file you would provide 
```
      parameters:
      - name: namespace
        value: cert-manager
```

Reply:
Here, this variable is different than `namespace` variable. This is used to have RoleBindings link to cert-manager's namespace services.

For these resources (roles and role-bindings), `kube-system` would be the value for `namespace`.

Reply:
got it - thanks for the clarification